src/developer/web-api/i18n.md

```diff
@@ -148,6 +148,8 @@ Send PUT request to `api/dataElements/{dataElementUID}` with full object payload
 }
 ```
 
+The status code will be `204 No Content` if the data value was successfully saved or updated, or `404 Not Found` if there was a validation error (e.g. more than one `SHORT_NAME` for the same `locale`).
```

Comment:
Do we really return 404 if there is an import conflict? I assume it should be 409?

Reply:
Just tested on play 2.36.11.1. API returned `404`

`PUT ` request to `https://play.dhis2.org/2.36.11.1/api/dataElements/fbfJHSPpUQD/translations` with the payload

`{
    "translations": [
        {
            "property": "SHORT_NAME",
            "locale": "en_GB",
            "value": "ANC 1st visit"
        },
        {
            "property": "NAME",
            "locale": "fr",
            "value": "Soin prénatal 1"
        },
        {
            "property": "NAME",
            "locale": "en_GB",
            "value": "ANC 1st visit"
        },
        {
            "property": "NAME",
            "locale": "en_GB",
            "value": "ANOTHER_VALUE (see that this is a duplicated NAME for en_GB)"
        }
    ]
}`

Reply:
A 409 conflict is returned for 2.37 onwards. 
```suggestion
The status code will be `204 No Content` if the data value was successfully saved or updated, or `409 Conflict` if there was a validation error (e.g. more than one `SHORT_NAME` for the same `locale`).
```